oxy/yaml/bms.yaml

```diff
@@ -0,0 +1,243 @@
+!Messages
+msgs: 
+#BMS BROADCAST
+- !CANMsg
+    id: "0x80"
+    desc: "accumulator status"
+    fields:
+    - !CANField
+        name: "BMS/Pack/Voltage"
+        unit: "V"
+        size: 2
+        decodings:
+        - !BigEndian
+            bits: 8
+        format: "high_voltage"
+    - !CANField
+        name: "BMS/Pack/Current"
+        unit: "A"
+        size: 2
+        decodings:
+        - !BigEndian
+            bits: 8
+        - !TwosComplement
+            bits: 16
+        format: "current"
+    - !CANField
+        name: "BMS/Pack/Amp-hours"
+        unit: "Ah"
+        size: 2
+        decodings:
+        - !BigEndian
+            bits: 8
+    - !CANField
+        name: "BMS/Pack/SOC"
+        unit: "%"
+        size: 1
+    - !CANField
+        name: "BMS/Pack/Health"
+        unit: "%"
+        size: 1
+
+- !CANMsg
+    id: "0x81"
+    desc: "BMS Status"
+    fields:
+    - !CANField
+        name: "BMS/State"
+        unit: ""
+        size: 1
+    - !CANField
+        name: "BMS/Faults"
+        unit: ""
+        size: 4
+        decodings:
+        - !LittleEndian
+            bits: 8
+    - !CANField
+        name: "BMS/Temps/Average"
+        unit: "C"
+        size: 1
+        decodings:
+        - !TwosComplement
+            bits: 8
+    - !CANField
+        name: "BMS/Temps/Internal"
+        size: 1
+        unit: "C"
+        decodings:
+        - !TwosComplement
+            bits: 8
+    - !CANField
+        name: "BMS/Cells/BurningStatus"
+        size: 1
+        unit: ""
+
+- !CANMsg
+    id: "0x82"
+    desc: "Shutdown Control"
+    fields:
+    - !CANField
+        name: "BMS/Shutdown/MPE"
+        size: 1
+        unit: ""
+
+- !CANMsg
+    id: "0x83"
+    desc: "Cell Data"
+    fields:
+    - !CANField
+        name: "BMS/Cells/Volts/High/Value"
+        size: 2
+        unit: "V"
+        decodings: 
+        - !LittleEndian
+            bits: 8
+        format: "cell_voltage"
+    - !CANField
+        name: "BMS/Cells/Volts/High/Chip"
+        size: 1
+        unit: ""
+        decodings: 
+        - !Half
+            bits: 4
+    - !CANField
+        name: "BMS/Cells/Volts/High/Cell"
+        index: 2
+        size: 1
+        unit: ""
+        decodings: 
+        - !Half
+            bits: 0
+    - !CANField
+        name: "BMS/Cells/Volts/Low/Value"
+        size: 2
+        index: 3
+        unit: "V"
+        decodings:
+        - !LittleEndian
+            bits: 8
+        format: "cell_voltage"
+    - !CANField
+        name: "BMS/Cells/Volts/Low/Chip"
+        index: 5
+        size: 1
+        unit: ""
+        decodings: 
+        - !Half
+            bits: 4
+    - !CANField
+        name: "BMS/Cells/Volts/Low/Cell"
+        index: 5
+        size: 1
+        unit: ""
+        decodings:
+        - !Half
+            bits: 0
+    - !CANField
+        name: "BMS/Cells/Volts/Ave/Value"
```

Comment:
Nit: I prefer `Avg` vs `Ave` for average. If its not average then idk what it is lol